types/models.d.ts

```diff
@@ -72,6 +72,8 @@ declare module 'mongoose' {
   };
 
   type UpdateWriteOpResult = mongodb.UpdateResult;
+  export type UpdateResult = mongodb.UpdateResult;
+  export type DeleteResult = mongodb.DeleteResult;
```

Comment:
```suggestion
  type UpdateResult = mongodb.UpdateResult;
  type DeleteResult = mongodb.DeleteResult;
```

to my knowledge adding `export` is unnecessary when using `declare module` (for example other types like `UpdateWriteOpResult` or `SaveOptions` can be imported without them being marked `export`).

this is minor, but consistency with the type above